en/docs/develop-components/develop-components-with-git.md

```diff
@@ -39,7 +39,7 @@ Authorizing Choreo as a GitHub application provides Choreo the following permiss
 
 ## Authorize Bitbucket with Choreo
 
-Authorizing Choreo as a Bitbucket application provides Choreo the following permissions to perform the respective actions on your behalf within the repository:
+Authorizing using a Personal Access Token (PAT) from Bitbucket provides Choreo with the following permissions to perform the respective actions on your behalf within the repository.
```

Comment:
In this case we don't need to capitalize the spelled-out version of PAT. The style guide recommends capitalizing the spelled-out version of the abbreviation only if the long form is a proper noun or is conventionally capitalized [1].
[1] https://developers.google.com/style/abbreviations#spelling-out

Reply:
Addressed in https://github.com/YashodhaSamarawickrama/docs-choreo-dev/pull/1